src/renderer/components/helpers/OpenQrUrl.tsx

```diff
@@ -172,7 +172,7 @@ export default async function processOpenQrUrl(
         openDialog(ConfirmationDialog, {
           message: tx(message, replacementValue),
           cb: resolve,
-          confirmLabel: tx('login'),
+          confirmLabel: tx('login_title'),
```

Comment:
Interestingly the `login` translation key did not exist. I'm temporarily using this one now. Should we introduce a `login` key or leave it like that?

Reply:
let's leave it as is - there are lots, esp. older strings, that do not follow our current default of "lowercase_text_with_underscores ~= key" - login_title seems one of the less weird examples here :)